modules/ding_test/ding_test.module

```diff
@@ -1,14 +1,15 @@
 <?php
+/**
+ * @file
+ * Support functionallity for testing.
+ */
 
 /**
  * Implements hook_xautoload().
  *
  * Have xautoload load classes from our vendor-dir.
- *
- * @param \Drupal\xautoload\Adapter\LocalDirectoryAdapter $adapter
- *   An adapter object that can register stuff into the class loader.
  */
-function ding_test_xautoload($adapter) {
```

Comment:
According to [Drupal Code standards](https://www.uptrends.com/) we should not user fully qualified class names in the code but add a `use` statement at the top of the file instead.